bin/generate-gutenboarding-design-thumbnails.mjs

```diff
@@ -12,24 +12,19 @@
  *
  * GENERATE THUMBNAILS:
  *
- * `node ./bin/generate-gutenboarding-design-thumbnails.mjs`
+ * `node ./bin/generate-gutenboarding-design-thumbnails.js`
  *
  * We should bump the version query param on the image src to cache bust the images too in client/landing/gutenboarding/available-designs.ts
```

Comment:
Shouldn't we update also this comment to mention the new path updated in https://github.com/Automattic/wp-calypso/pull/51306/commits/66c4677e0c3c52374e3d1b51bfd316eb5954bb3d ?

Reply:
Updated in https://github.com/Automattic/wp-calypso/pull/51306/commits/3e193ad66194537db7383f33c91352547199045f